docs/squash-api/index.rst

```diff
@@ -0,0 +1,86 @@
+##########
+squash-api
+##########
+
+.. list-table::
+   :widths: 10,40
+
+   * - Edit on GitHub
+     - `/services/squash-api <https://github.com/lsst-sqre/lsp-deploy/tree/master/services/squash-api>`__
+   * - Type
+     - Helm_
+   * - Namespace
+     - ``squash-api``
+
+.. rubric:: Overview
+
+The ``squash-api`` app deploys a REST API for storing and visualizing Science Pipelines metrics. You can learn more about SQuaSH in SQR-009_.
+
+.. _SQR-009: https://sqr-009.lsst.io/
+
+Currently, the ``squash-api`` is deployed using the ``squash-sandbox`` and ``squash-prod`` environments along with other services:
+
+- argo-cd
+- cert-issuer
+- cert-manager
+- chronograf
+- gafaelfawr
+- influxdb
+- kapacitor
+- ingress-nginx
+- vault-secrets-operator
+
+You can reach the following services, for example, for the ``https://squash-sandbox.lsst.codes`` deployment:
+
+- https://squash-sandbox.lsst.codes (SQuaSH API)
+- https://squash-sandbox.lsst.codes/argo-cd  (Argo CD)
+- https://squash-sandbox.lsst.codes/chronograf (Chronograf)
+- https://squash-sandbox.lsst.codes/influxdb (InfluxDB)
+
+The Science Pipelines use lsst.verify_ to collect metrics and produce verification jobs that are uploaded to the SQuaSH API. An internal task in the SQuaSH API extracts metrics and metadata from the verification jobs and store them in InfluxDB.
```

Comment:
collect metrics --> collect metrics and their measurements
extracts metrics and metadata --> extracts metrics, measurements and metadata
store them --> stores them